### PR TITLE
address list error fix

### PR DESCRIPTION
**Pull request by atedros-ch**

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/main/controllers/features/SoleTrader/soleTraderCorrespodanceAddressDetailsController.ts [hunk 1]
```diff
@@ -4,6 +4,8 @@ import { validationResult } from "express-validator";
 import { FormattedValidationErrors, formatValidationError } from "../../../validation/validation";
 import { selectLang, addLangToUrl, getLocalesService, getLocaleInfo } from "../../../utils/localise";
 import { SOLE_TRADER_AUTO_LOOKUP_ADDRESS_LIST, SOLE_TRADER_AUTO_LOOKUP_ADDRESS, SOLE_TRADER_CORRESPONDENCE_ADDRESS_CONFIRM } from "../../../types/pageURL";
+import { getCountryFromKey } from "../../../utils/web";
+import { log } from "console";
```
Comment (swalke-ch) [inline src/main/controllers/features/SoleTrader/soleTraderCorrespodanceAddressDetailsController.ts:8]: where are we using this?

File: locales/en/corresspondence-address-lookUp.json [hunk 2]
```diff
@@ -10,6 +10,7 @@
 
     "error-correspondenceLookUpAddressNoAddress" : "Select the correspondence address",
     "error-correspondenceLookUpAddressNoPostCode" : "Enter a postcode",
+    "error-postCodeYouEnterDoesntExist" : "You enter a postcode that doesn't exist",
```
Comment (swalke-ch) [inline locales/en/corresspondence-address-lookUp.json:13]: Is corresponding welsh translation added?

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by sardhani-ch — APPROVED** [review]

**Pull request merged**